src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Arm extends SubsystemBase {
+  public Arm() {}
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
+
+  public void highMidPos(){}
+  public void lowPos(){}
```

Comment:
I'm assuming the highMidPos() and lowPos() methods move the arm to those respective positions. This isn't necessary but I personally think the code would be more clear if they were renamed to something like "moveToLowPos()" or "setLowPos()" kind of like how the 2023 code had a method called "setScoringInProgress()". This isn't a necessary change though if other people disagree.